src/renderer/fonts/font_loader.rs

```diff
@@ -158,3 +165,23 @@ fn font_style(bold: bool, italic: bool) -> FontStyle {
         (false, false) => FontStyle::normal(),
     }
 }
+
+fn font_hinting(hinting: &str) -> FontHinting {
+    let hinting = hinting.to_lowercase();
+    match hinting.as_str() {
+        "full" => FontHinting::Full,
+        "normal" => FontHinting::Normal,
+        "slight" => FontHinting::Slight,
+        _ => FontHinting::None
+    }
+}
+
+fn font_edging(edging: &str) -> Option<Edging> {
+    let edging = edging.to_lowercase();
+    match edging.as_str() {
+        "alias" => Some(Edging::Alias),
+        "antialias" => Some(Edging::AntiAlias),
+        "subpixelantialias" => Some(Edging::SubpixelAntiAlias),
+        _ => None
+    }
+}
```

Comment:
Maybe implementing `ParseFromValue` like it's done in the cursor renderer with `VfxMode` would make this less messy and also allow including the enum _directly_ in the struct, without a potentially broken invariant.

Reply:
Updated with `ParseFromValue`, I don't think that it looks better. But good to have consistency.